charts/fluid-dataloader/jindocache/CHANGELOG.md

```diff
@@ -18,7 +20,7 @@
 ### 0.2.0
 - Support JindoFSx
 
-### 0.1.0
+### 0.2.0
```

Comment:
Why do you need to change 0.1.0  to 0.2.0?

Reply:
Accidentally modified，done